src/components/ProductCard.js

```diff
@@ -1,21 +1,46 @@
 import "./ProductCard.css";
-import Niacinamide from "../images/niacinamide-bottle.png";
+import { useEffect, useState } from "react";
+
+export default function ProductCard({ name, image, url }) {
+  const [classForImage, setClassForImage] = useState("");
+
+  useEffect(() => {
```

Comment:
You might consider creating in an object to "map" the ingredients to the respective class. Or extend your data by some kind of category ("creme", "bottle", "powder" ...) to consolidate the data instead of defining the condition by name. Then you could use this category to assign an image class (and maybe other things, if needed).
You'll get a better readability from that, cleaner code and avoid repetition.

Reply:
Great idea, i will implement a new category to differentiate between the packaging and clean up the code. Thank you ☺️